feature_engine/encoding/base_encoder.py

```diff
@@ -144,8 +144,16 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
 
         # check if NaN values were introduced by the encoding
         if X[self.encoder_dict_.keys()].isnull().sum().sum() > 0:
+            # obtain the name(s) of the columns have null values
+            nan_columns = X.columns[X.isnull().any()].tolist()
+            if len(nan_columns) > 1:
+                nan_columns_str = ", ".join(nan_columns)
+            else:
+                nan_columns_str = nan_columns[0]
+
             warnings.warn(
-                "NaN values were introduced in the returned dataframe by the encoder."
+                f"NaN values were introduced in the feature(s) titled "
```

Comment:
Could we re-phrase this error sentence to this:

f"NaN values were introduced in the feature(s) {nan_columns_str} during the encoding."

In other words could you please remove "titled"?

Reply:
done.